threadit-web/src/api/SpoolAPI.ts

```diff
@@ -70,4 +77,92 @@ export default class SpoolAPI {
 
         return await response.json();
     }
+
+    static async getAllMods(spoolId: string): Promise<IUserProfile[]> {
+        const response = await get(allModsForSpoolEndpoint + spoolId);
+
+        if (!response.ok) {
+            throw new Error(`Failed to get all mods for the spool: ${await response.text()}`);
+        }
+
+        return await response.json();
+    }
+
+    static async removeModerator(spoolId: string, userId: string): Promise<IUserProfile> {
+        const response = await get(removeModeratorEndpoint + spoolId + '/' + userId);
```

Comment:
if you add authentication to these endpoints you will need to change `get` to `getWithAuth` (for most of these)